src/PHPUnit/Constraint/Page.php

```diff
@@ -22,36 +40,30 @@ public function __construct($string, $uri = '')
      *
      * @return bool
      */
-    protected function matches($other) : bool
+    protected function matches($other): bool
     {
         $other = $this->normalizeText($other);
-        return mb_stripos($other, $this->string, null, 'UTF-8') !== false;
+        return mb_stripos($other, $this->string, 0, 'UTF-8') !== false;
     }
 
-    /**
-     * @param $text
-     * @return string
-     */
-    private function normalizeText($text)
+    private function normalizeText(string $text): string
     {
         $text = strtr($text, "\r\n", "  ");
         return trim(preg_replace('/\\s{2,}/', ' ', $text));
     }
 
     /**
      * Returns a string representation of the constraint.
-     *
-     * @return string
      */
-    public function toString() : string
+    public function toString(): string
     {
         return sprintf(
             'contains "%s"',
             $this->string
         );
     }
 
-    protected function failureDescription($pageContent) : string
+    protected function failureDescription($pageContent): string
```

Comment:
parameter is a string.

Reply:
```diff
+    /**
+     * @param string $pageContent
+     * @return string
+     */
    protected function failureDescription($pageContent): string
```

Reply:
Making the change you suggest also generates errors in the tests:

```shell
 1) PhpBrowserTest: See text after js comparision operator
 Test  tests/unit/Codeception/Module/PhpBrowserTest.php:testSeeTextAfterJSComparisionOperator
                                                                                                                                                                                                                                                                                          
  [PHPUnit\Framework\Exception] Declaration of Codeception\PHPUnit\Constraint\Page::failureDescription(string $pageContent): string should be compatible with PHPUnit\Framework\Constraint\Constraint::failureDescription($other): string at ../../../src/PHPUnit/Constraint/Page.php:65  
```